test/sass/import-css.sass

```diff
@@ -1,7 +1,7 @@
 // Special behavior of node-sass/libsass with CSS-files
 // 1. CSS-files are not included, but linked with @import url(path/to/css)
-@import ../node_modules/css/some-css-module.css
+@import ~css/some-css-module.css
```

Comment:
Thanks for spotting this. Seems like it slipped through with some of the last commits...